src/main/kotlin/app/batch/HbaseClient.kt

```diff
@@ -49,5 +49,4 @@ open class HbaseClient(
         }
     }
 
-    fun close() = connection.close()
```

Comment:
Is that really superfluous?

Reply:
Hmm that was probably an IntelliJ code inspection change. I'll put a call to it after the pipeline has run though.